package.json

```
@@ -28,9 +28,24 @@
             {
                 "command": "local-history.viewAllForActiveEditor",
                 "title": "Local History: Active Editor"
+            },
+            {
+                "command": "local-history.revertActiveEditorToPrevRevision",
+                "title": "Local History: Revert to Previous Revision",
+                "icon": {
```

You can do the following so there's no need to include the icons:

```suggestion
                "icon": "$(history)"
```